script.js

```diff
@@ -1,17 +1,49 @@
-$('.save-btn').on('click', createIdea);
+$('.save-btn').on('click', createCard);
 $('.new-idea-section').on('click', deleteIdea);
+$('.new-idea-section').on('click', upvoteIdea);
+$('.new-idea-section').on('click', downvoteIdea);
 
-function createIdea(event) {
+
+recallCards();
+
+function upvoteIdea(event) {
+  if ($(event.target).siblings('.quality').text() === 'Quality: swill') {
```

Comment:
Apply the DRY principle here. assign this (`$(event.target).siblings('.quality')) to a var so you can use that and not have to type it out everytime